openid-data-classes/src/commonMain/kotlin/at/asitplus/openid/AuthenticationRequestParameters.kt

```diff
@@ -258,6 +266,82 @@ data class AuthenticationRequestParameters(
      */
     @SerialName("code_challenge_method")
     val codeChallengeMethod: String? = null,
+
+    //CSC additionally defines the following parameters
```

Comment:
Imho this comment will never be read as it does not show up in the generated documentation for each property below.